src/stripeJavaLanguageClient/javaServerStarter.ts

```diff
@@ -110,31 +111,42 @@ function prepareParams(
     );
     params.push(path.resolve(__dirname, javaServerPath, configDir));
   } else {
-    params.push(resolveConfiguration(context, configDir));
+    const config = resolveConfiguration(context, configDir);
+    if (config) {
+      params.push(config);
+    } else {
+      return 'Failed to get server configuration file.';
+    }
   }
+
   params.push('-data');
   params.push(workspacePath);
+
   return params;
 }
 
-function resolveConfiguration(context: ExtensionContext, configDir: string) {
+export function resolveConfiguration(context: ExtensionContext, configDir: string): string {
   ensureExists(context.globalStoragePath);
-  const extensionPath = path.resolve(context.extensionPath, 'package.json');
-  const packageFile = JSON.parse(fs.readFileSync(extensionPath, 'utf8'));
-  let version;
-  if (packageFile) {
-    version = packageFile.version;
-  } else {
-    version = '0.0.0';
+  let version = '0.0.0';
+  try {
+    const extensionPath = path.resolve(context.extensionPath, 'package.json');
+    const packageFile = JSON.parse(fs.readFileSync(extensionPath, 'utf8'));
+    if (packageFile) {
+      version = packageFile.version;
+    }
+  } catch {
+    console.log('Cannot locate package.json');
```

Comment:
same question re:output channel?

Reply:
same reason to why i moved out the other one. I dont have a preference either :). i will move it to output channel + telemetry